### ISSUE TITLE
Loading Pretrained BERT Model

### ISSUE THREAD

**Issue by g-luo** [facebookresearch/mmf#606]
## ❓ Questions and Help

Hello, I am trying to fine tune a text only BERT model.

I was wondering if there was a model in the model zoo I could use--I could only find a unimodal text model finetuned on hateful memes. I was also having trouble loading Base BERT given by Google since it has a ckpt.data-000... and ckpt.index file, and I wasn't sure how to load them.

For context, I'm just trying to run the command 
`MMF_USER_DIR="." mmf_run config="./configs/text_foil_experiment_config.yaml" \
        model=unimodal_text \
        dataset=foil \
        run_type=train_val \
        training.num_workers=0 \
        checkpoint.resume_file="./base_bert/bert_model.ckpt" \
        checkpoint.resume_pretrained=True`

My code is also at this Github url: https://github.com/g-luo/foil_mmf

Thanks!

**Comment by apsdehal** [comment]
Hi,

If you use unimodal_text model and include `configs/models/unimodal/bert.yaml` in your experiment config, you would get a text bert model initialized from bert-base-uncased checkpoint from Google. Let me know if something isn't clear in how to use it now.

**Comment by g-luo** [comment]
Thanks! For some reason I'm getting this error when I try running fine tune on the bert model--do you have any advice for where this is coming from?
<img width="571" alt="Screen Shot 2020-10-05 at 10 33 50 AM" src="https://user-images.githubusercontent.com/36020488/95112842-54d68b00-06f6-11eb-99bd-26b705f95e49.png">

**Comment by g-luo** [comment]
Additionally, I had a question related to prediction -- I'm running prediction and I'm a little confused about the output files. I just see a bunch of new folders in the save directory with nothing in them. (eg one folder is names foil_visual_bert_9859962/reports) and there is nothing inside. I'm running the following command:
`
MMF_USER_DIR="." mmf_predict config="./configs/foil_experiment_config.yaml" \
    model=visual_bert \
    dataset=foil \
    run_type=test \
    checkpoint.resume_file="./save/visual_bert_final.pth" \
    checkpoint.resume_pretrained=False \
    training.num_workers=0
`

**Comment by g-luo** [comment]
Another question: why does running mmf_run show a counter that's lower than the size of my total test set? ie running `MMF_USER_DIR="." mmf_run config=./configs/text_foil_experiment_config.yaml \
        model=unimodal_text \
        dataset=foil \
        run_type=test \
        checkpoint.resume_file="./save/unimodal_text_final.pth" \
        checkpoint.resume_pretrained=False` shows x / the 6218 when my test set size has 198960 items.

**Comment by apsdehal** [comment]
Yes, you are right. The counter is specifically for the number of batches.
